frameworks/render-stacktrace.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
-var hljs = require('./hl.js');
+// var hljs = require('./hl.js');
```

Comment:
The `highlight.js` api was improved and can now be required directly without the funky stuff in `hl.js`